src/adapters/appnexusAst.js

```diff
@@ -193,8 +195,11 @@ function AppnexusAstAdapter() {
       if (type === 'native') bid.mediaType = 'native';
       if (type === 'video') bid.mediaType = 'video';
       if (type === 'video-outstream') bid.mediaType = 'video-outstream';
-      const placement = bidRequests[bid.adId].placementCode;
-      bidmanager.addBidResponse(placement, bid);
+
+      if (bid.adId in bidRequests) {
```

Comment:
Yes, if the response from the 1st auction is extremely slow and arrives after the 2nd auction has already started and `bidRequests` list has been emptied, then bidRequests will have adId's from the second auction, and adId from the first auction will have no corresponding entry.